### PR TITLE
Big bundle of UI customizations for OHC

### PR DESCRIPTION
**Pull request by abbistani**

### CODE REVIEW

**Comment by mark-cooper** [comment]
@abbistani all of these changes are OHC specific? There's nothing for the anthro ui?

**Comment by abbistani** [comment]
Yep.  All OHC specific, and nothing in the app layer.

**Comment by mark-cooper** [comment]
@ray-lee everything look reasonable to you? Gonna run it locally now but would appreciate you taking a look also 😄

File: src/plugins/recordTypes/collectionobject/forms/index.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 
 export default configContext => ({
   default: defaultForm(configContext),
-  archeology: archeolgyForm(configContext),
+  'archeology-parent': archeolgyParentForm(configContext),
+  'archeology-child': archeolgyChildForm(configContext),
   history: historyForm(configContext),
   naturalhistory: naturalHistoryForm(configContext),
+  inventory: inventoryForm(configContext),
+  photo: photoForm(configContext),
```
Comment (ray-lee) [inline src/plugins/recordTypes/collectionobject/forms/index.js]: Same as above. If the intent is only to hide this form,
```
photo: {
  disabled: true,
},
```

File: src/plugins/recordTypes/collectionobject/forms/inventory.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  disabled: true,
+  messages: defineMessages({
+    name: {
+      id: 'form.collectionobject.inventory.name',
+      defaultMessage: 'Inventory Template',
+    },
+  }),
+  sortOrder: 1,
+  template: template(configContext),
+});
```
Comment (ray-lee) [inline src/plugins/recordTypes/collectionobject/forms/inventory.jsx]: If all we're doing is hiding this form, it's enough to add `disabled: true`. Don't need to reiterate messages, sortOrder, and template. And `disabled: true` is short enough that I don't think it's worth breaking out into a separate file, so I would put it directly in forms/index.js.

File: src/plugins/recordTypes/collectionobject/forms/photo.jsx [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+  disabled: true,
+  messages: defineMessages({
+    name: {
+      id: 'form.collectionobject.photo.name',
+      defaultMessage: 'Photograph Template',
+    },
+  }),
+  sortOrder: 2,
+  template: template(configContext),
+});
```
Comment (ray-lee) [inline src/plugins/recordTypes/collectionobject/forms/photo.jsx]: Same as inventory.jsx. `disabled: true` is enough to hide this form.

File: src/plugins/recordTypes/collectionobject/optionLists.js [hunk 4]
```diff
@@ -379,6 +380,10 @@ export default {
         id: 'option.recordStatuses.ready-for-review',
         defaultMessage: 'ready for review',
       },
+      'revision-needed': {
+        id: 'option.recordStatuses.revision-needed',
+        defaultMessage: 'revision needed',
+      },
       temporary: {
         id: 'option.recordStatuses.temporary',
         defaultMessage: 'temporary',
```
Comment (ray-lee) [inline src/plugins/recordTypes/collectionobject/optionLists.js]: Here it looks like we've added two values to the list, ready-for-review and revision-needed. So it's good that there are messages for them. Those messages will get merged in to the existing ones from cspace-ui. But we're also redefining messages for approved, in-process, new, and temporary. Those will also get merged into the existing messages, but they're exactly the same as the existing messages, so they have no effect. In general it's better to keep your plugin as small as possible -- everything should be additional config, or an override to existing config that makes a change. That makes it faster to download/parse/merge the plugin, and makes it easy to see exactly what the plugin does.

**Comment by mark-cooper** [comment]
@abbistani @ray-lee this was merged but not deployed. @abbistani will make a new branch and bump the version again for the corrections. There will also be app layer config update for the repeating field -- any data concerns / are they using the field currently?

**Pull request merged**